content/hi/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,113 @@
+---
+title: क्रॉनजॉब के साथ स्वचालित कार्य चलाना
+min-kubernetes-server-version: v1.21
+reviewers:
+- chenopis
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+यह पेज दिखाता है कि कैसे Kubernetes {{< glossary_tooltip text="CronJob" term_id="cronjob" >}} ऑब्जेक्ट का उपयोग करके स्वचालित कार्य चलाए जाते हैं।
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## क्रॉनजॉब बनाना {#creating-a-cron-job}
+
+क्रॉन जॉब्स को एक कॉन्फ़िग फ़ाइल की आवश्यकता होती है।
+यहाँ एक क्रॉनजॉब के लिए मैनिफेस्ट है जो हर मिनट एक साधारण डेमो कार्य चलाता है:
+
+{{% code_sample file="application/job/cronjob.yaml" %}}
+
+इस कमांड का उपयोग करके उदाहरण क्रॉनजॉब चलाएं:
+
+```shell
+kubectl create -f https://k8s.io/examples/application/job/cronjob.yaml
+```
+आउटपुट इस तरह का होगा:
+
+```
+cronjob.batch/hello created
+```
+
+क्रॉनजॉब बनाने के बाद, इस कमांड का उपयोग करके इसकी स्थिति प्राप्त करें:
+
+```shell
+kubectl get cronjob hello
+```
+
+आउटपुट इस तरह का होगा:
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        <none>          10s
+```
+
+जैसा कि आप कमांड के परिणामों से देख सकते हैं, क्रॉन जॉब ने अभी तक कोई जॉब शेड्यूल या चलाया नहीं है।
```

Comment:
You have used शेड्यूल here and स्केड्यूलर in another PR. IME, शेड्यूल is the more commonly used pronunciation.

Reply:
Resolve the suggestion @divya-mohan0209